allure-generator/src/main/javascript/components/testresult/TestResultView.js

```diff
@@ -15,6 +14,7 @@ const SEVERITY_ICONS = {
 };
 
 @behavior('TooltipBehavior', {position: 'left'})
+@behavior('ClipboardBehavior', {position: 'left'})
```

Comment:
How about built this option into behavior itself? It is not about copying and will never be changed